pkg/state/state_test.go

```diff
@@ -0,0 +1,32 @@
+package state_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"helm.sh/helm/v3/pkg/chart"
+
+	"github.com/openshift-psap/special-resource-operator/pkg/state"
+)
+
+func TestState(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "State Suite")
+}
+
+var _ = Describe("State", func() {
+	Describe("GenerateName", func() {
```

Comment:
you don't need second Describe, It can be directly under first Describe

Reply:
Removed the top-level `Describe`.